server/routerlicious/packages/services-core/src/document.ts

```diff
@@ -87,6 +87,9 @@ export interface IDeliState {
 
     // List of successfully started lambdas at session start
     successfullyStartedLambdas: LambdaName[];
+
+    // Timestamp of lastCheckpoint
+    checkpointTimestampEpoch: number;
```

Comment:
>number [](http://example.com/codeflow?start=30&length=6)

Should we use date instead?

Reply:
Discussed offline, use number to align with other timestamp format usage in the codebase.